### PR TITLE
Add performance test based on locust.io

### PR DESCRIPTION
**Pull request by lebdron**
### Description of the Change
Performance test that sends a transaction using asynchronous API and waits until the final status - committed or rejected - is received from the status stream.

### How to run
[README.md](https://github.com/lebdron/iroha/blob/7426dc869654076b2d8c37cd74d4add6d9ffde9f/test/load/README.md)
>Specify locustfile in `LOCUSTFILE_PATH` in Compose file

### CODE REVIEW

File: test/load/locustfile-performance.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+import time
+import binascii
+
+from locust import Locust, TaskSet, events, task
+
+import grpc.experimental.gevent as grpc_gevent
+grpc_gevent.init_gevent()
+
+import grpc
+from irohalib import Iroha, IrohaGrpc
```
Comment (igor-egorov) [inline test/load/locustfile-performance.py]: Let's move to Iroha from pypi.
```
pip install iroha
```
and
```
from iroha import Iroha, IrohaGrpc, IrohaCrypto
```

The second preparation step from test/load/README.md will be much simpler.

**Comment by neewy** [comment]
Encountered the following problem:
```
ERROR: yaml.constructor.ConstructorError: while constructing a mapping
  in "./docker-compose.yml", line 18, column 7
expected a mapping or list of mappings for merging, but found scalar
  in "./docker-compose.yml", line 6, column 16
```

**Comment by neewy** [comment]
The problem still persists on macOS and Ubuntu 18.10 x64

**Comment by neewy** [comment]
Ubuntu
docker-compose version 1.22.0, build unknown
Docker version 18.09.2, build 6247962

**Comment by neewy** [comment]
macOS
Docker version 18.09.1, build 4c52b90
docker-compose version 1.23.2, build 1110ad01

**Pull request merged**